src/templates/ratings/add.html

```diff
@@ -7,6 +7,11 @@
           {% endif %}>
       {% if isModal %}
         <h3>{{ existingReview and _('Edit Review') or _('Leave a Review') }}</h3>
+	<p class="reviews--guideline">
+          {{ _('Please read the <a href="{url}" target="_blank">Review Guidelines</a> for more details about rating apps.
+           Reviews that do not meet these guidelines may be removed by our
```

Comment:
It's nice to line this indent up so the the string starts in the same place as the last line:

```
'Please read...
 Reviews that do not...
```